examples/animation/double_pendulum_animated.py

```diff
@@ -86,5 +97,4 @@ def animate(i):
 ani = animation.FuncAnimation(fig, animate, np.arange(1, len(y)),
                               interval=25, blit=True, init_func=init)
 
-#ani.save('double_pendulum.mp4', fps=15)
```

Comment:
yes

On Thu, May 26, 2016 at 9:55 PM, Ashley DaSilva <notifications@github.com>
wrote:

> In examples/animation/double_pendulum_animated.py
> <https://github.com/matplotlib/matplotlib/pull/6486#discussion_r64848024>:
>
> > @@ -86,5 +97,4 @@ def animate(i):
> >  ani = animation.FuncAnimation(fig, animate, np.arange(1, len(y)),
> >                                interval=25, blit=True, init_func=init)
> >
> > -#ani.save('double_pendulum.mp4', fps=15)
>
> Sure - I see how that could be useful. Keep as a comment?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6486/files/0bd23dcdf6e696b67318eab42fabfff0be41e1b9#r64848024>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARy-NhveMgFHXFuPt8O6sicISR_Y-7Nks5qFk8hgaJpZM4IoFO_>
> .
>